third_party/move/move-compiler/src/parser/ast.rs

```diff
@@ -1312,7 +1317,7 @@ impl AstDebug for StructDefinition {
         w.write(&format!("struct {}", name));
         type_parameters.ast_debug(w);
         match layout {
-            StructLayout::Singleton(fields) => w.block(|w| {
+            StructLayout::Singleton(fields, _) => w.block(|w| {
```

Comment:
We can use that to print the un-desugared struct definition. But since it's for debug use, I think it's ok or even preferred to print the desugared definition.